app/views/site/_study_download_data.html.erb

```diff
@@ -38,6 +38,9 @@
                 <%= link_to "<i class='fas fa-download'></i> Get download command for all study data".html_safe,
                             '#/', class: 'btn btn-default get-download-command', id: 'get-download-command__all' %></p>
               <% if @directories.any? %>
+                <p class="lead command-container" id="command-container-nodirs">
+                  <%= link_to "<i class='fas fa-download'></i> Get download command for study files only (no folders)".html_safe,
+                              '#/', class: 'btn btn-default get-download-command', id: 'get-download-command__nodirs' %></p>
```

Comment:
I'm not sure users will understand why they would/wouldn't want folders.  This sounds like a good demo day discussion